sdk/core/core-https/package.json

```diff
@@ -52,7 +52,7 @@
     "test:node": "npm run build:test:node && npm run unit-test:node && npm run integration-test:node",
     "test": "npm run clean && npm run build:ts && npm run bundle:test:node && npm run unit-test:node && npm run bundle:test:browser && npm run unit-test:browser && npm run integration-test:node && npm run integration-test:browser",
     "unit-test:browser": "karma start --single-run",
-    "unit-test:node": "mocha --require source-map-support/register --reporter ../../../common/tools/mocha-multi-reporter.js dist-test/index.node.js",
+    "unit-test:node": "mocha --require source-map-support/register --timeout 20000 --reporter ../../../common/tools/mocha-multi-reporter.js dist-test/index.node.js",
```

Comment:
can we avoid needing such a long timeout? I've been trying to keep the tests as fast as possible

Reply:
I'll use the time bending trick you pointed at below! That was awesome 🌞